### ISSUE TITLE
ListView in Windows Mobile crashes when referencing WinUI Library

### ISSUE THREAD

**Issue by sonnemaf** [Microsoft/microsoft-ui-xaml#10]
If you have a ListView in a UWP app and run it on Windows Mobile it crashes with the following UnhandledException:

Windows.UI.Xaml.Markup.XamlParseException 
Cannot deserialize XBF metadata type list as 'RevealListViewItemPresenter' was not found in namespace 'Microsoft.UI.Xaml.Controls'. [Line: 0 Position: 0]

MainPage.xaml
``
<Page x:Class="App3.MainPage"
      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
      xmlns:local="using:App3"
      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
      Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
      mc:Ignorable="d">
    <Grid>
        <ListView>
            <ListViewItem Content="aaa" />
            <ListViewItem Content="bbb" />
            <ListViewItem Content="ccc" />
            <ListViewItem Content="ddd" />
        </ListView>
    </Grid>
</Page>
``

App.xaml
``
<Application x:Class="App3.App"
             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
             xmlns:local="using:App3">
    <Application.Resources>
        <XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" />
    </Application.Resources>
</Application>
``

**Steps to reproduce the bug**
Create an UWP app. Add WinUI using NuGet. Add a XamlControlsResources to the App.xaml as described in the readme.txt. Add a ListView with some ListViewItems to the Grid of the MainPage.xaml. Run the app on Windows Mobile (I used the Mobile Emulator 10.0.15254.0 1080p). 

**Expected behavior**
An app running on Windows Mobile, no crash

**Screenshots**

**Version Info**
TargetPlatformVersion 10.0.17763.0
TargetPlatformMinVersion 10.0.15063.0

NuGet package version: 
Microsoft.UI.Xaml 2.0.181018003

Windows 10 version:
1809 - 17763.55

Device form factor:
- Mobile

**Additional context**

**Comment by jevansaks** [comment]
Thanks for the report! We will investigate and report back.

**Comment by licanhua** [comment]
sonnemaf, this problem is already resolved in Microsoft.UI.Xaml 2.1.181025003.
You should be using X86 when you build your project, so you have two options:
1. build and deploy your project with x64
2. or update the NuGet package

**Comment by jevansaks** [comment]
I think this might be an issue with the framework package, @licanhua could you try with the non-prerelease package? We didn't take any fixes into the 2.1 fork that would have changed this behavior.

**Comment by jevansaks** [comment]
We've checked in a fix, it will be in the next release.

**Comment by EmaginationStore** [comment]
I am using Microsoft.UI.Xaml version 2.0.181018003.1 and have received the same error when displaying a listview on Windows 10 Mobile Enterprise 1703 - 15063.1563.

**Comment by jevansaks** [comment]
Sorry, I misspoke before, the fix did not make it into the 2.0 branch. It will be in 2.1. I don't have a good workaround for you but if you put something like this in your app.xaml it should elide the issue (though you will stop getting reveal on ListViewItems on mobile, but that's probably ok):

```xaml
       <Style TargetType="ListViewItem" x:Key="ListViewItemRevealStyle"/>
```

Basically by having this style be empty it should override the one from XamlControlsResources.
